templates/quickstart-crowd-dc-with-vpc.template.yaml

```diff
@@ -507,20 +507,17 @@ Parameters:
     ConstraintDescription: Must be the name of an existing Route53 Hosted Zone.
     Description: The domain name of the Route53 PRIVATE Hosted Zone in which to create cnames
     Type: String
-  JiraProduct:
+  CrowdProduct:
```

Comment:
I think this parameter is unnecessary. We only have it for Jira because there are multiple Jira editions which can be installed with the same Quick Start. Crowd only has one edition.

Reply:
Yup its redundant, I'll remove it